codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/credentialresolverservices/IdentityProvidingSSOClientIntegration.kt

```diff
@@ -8,6 +8,7 @@ import software.amazon.smithy.swift.codegen.core.SwiftCodegenContext
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
```

Comment:
Changes here on out are just codegen changes that generate results you saw above.